Client/src/components/Layout/Desktop/index.js

```diff
@@ -1,34 +1,33 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
```

Comment:
use single quotation 
```suggestion
import React, { Component } from 'react';
```

Reply:
@naremanhilles  check your eslint